modules/mixed-arch-modify-machine-set.adoc

```diff
@@ -0,0 +1,73 @@
+//Module included in the following assembly
+//
+//post_installation_configuration/cluster-tasks.adoc
+
+:_content-type: PROCEDURE
+[id="mixed-arch-modify-machine-set_{context}"]
+
+= Adding the `arm64` bootimage to your machine set
```

Comment:
@Prashanth684 @aleskandro I just fixed the formating that'll help based on Alessandros comments about the machine set:
It was recommended to me, to make this process its own section (Prashanth, kinda what we were talking about in 1-1). Since "Configuring a heterogeneous cluster" is now its own section, the deploying is much easier to follow. 
Id rather have the 2nt section be modifying the machine set and adding the `arm64` image as its own procedure, as it's now easier to follow for the whole deployment process.

Reply:
Yes i agree that it should be called out as a separate section, but  i was thinking more of a hybrid approach. We link to the creating machinesets doc and just give an example here with the resource id. @kelbrown20 you could use the same machineset referenced in the adding machinesets doc and just modify the resource id. does that sound reasonable?

Reply:
Yeah I was thinking of that as well. Where its its own procedure but I can reference the creating machinesets doc. Lemme try to jot something up